app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
```

Comment:
tìm hiểu và dùng I18n nhé 

search `I18n in rails ` co hướng dẫn nè

Reply:
vâng ạ, cảm ơn chị